src/igvfd/schemas/changelogs/matrix_file.md

```diff
@@ -2,6 +2,9 @@
 
 ### Minor changes since schema version 7
 
+* Extend `principal_dimension` enum list to include `cell barcode`, `singlet cell` and `filtered singlet, sgRNA positive cell`.
+* Extend `secondary_dimensions` enum list to `sgRNA` and `umi count`.
```

Comment:
Technically it should be UMI instead of umi (as UMI = unique molecular identifiers). https://www.illumina.com/techniques/sequencing/ngs-library-prep/multiplexing/unique-molecular-identifiers.html

Reply:
done